tests/Compressor/HtmlCompressorTest.php

```diff
@@ -6,19 +6,22 @@
 use WyriHaximus\HtmlCompress\Compressor\CompressorInterface;
 use WyriHaximus\HtmlCompress\Compressor\HtmlCompressor;
 
+/**
+ * @internal
+ */
 final class HtmlCompressorTest extends TestCase
 {
     /** @var CompressorInterface */
     private $compressor;
 
-    public function setUp()
+    protected function setUp()
```

Comment:
It will cause error when running unit tests with `PHPUnit` `6.2`.

Please look at related comment on this PR #47  to know about more details.